build/scripts/cleanup_perf.ps1

```diff
@@ -17,21 +18,37 @@ try {
 }
 catch {}
 
-# Move all etl files to the a folder for archiving
-echo "creating ToArchive directory"
-mkdir ToArchive
-echo "moving C:/CPC/DataBackup* to ToArchive"
-mv C:/CPC/DataBackup* ToArchive
-ls ToArchive
+try {
+    C:/CPC/cpc.exe /Stop /SkipClean 2>&1 | Out-Null
```

Comment:
Even if CPC is hard coded now, eventually it will be a NuGet.  We should factor this out now into a parameter / global.  Today it will have the value of the hard coded location and once it moves to a NuGet it will be a simple change.